modules/nw-ingress-configuring-application-domain.adoc

```diff
@@ -5,7 +5,7 @@
 
 :_content-type: PROCEDURE
 [id="nw-ingress-configuring-application-domain_{context}"]
-= Configuring application domain for the Ingress Controller Operator
+= Configuring cluster domain for applications using appsdomain option.
```

Comment:
You could change "appsdomain option" to "the appsDomain option".

Reply:
Alternatively, you could make the heading a little more explicit, as follows:

> "Specifying an alternative cluster domain using the appsDomain option"

Reply:
Acked